install/ui/util/make-jquery.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
+
+if [ $# -ne 1 ]; then
+    echo 'The script requires exactly one argument (a jQuery version):'
+    echo
+    echo '  $ ./make-jquery.sh 3.4.1'
+    echo
+    exit 1
+fi
+
+WD=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+JQUERY_VERSION=$1
+
+# Apply patches
+cd /tmp
```

Comment:
Please use `mktemp` to create a temporary directory instead. Record its name and remove afterwards.

Reply:
Thanks for the remark. Fixed.